src/sentry/static/sentry/app/views/organizationDashboard.jsx

```diff
@@ -215,7 +216,7 @@ const ProjectListOld = createReactClass({
                     {project.isBookmarked && (
                       <span className="bookmark icon-star-solid" />
                     )}
-                    {project.name}
+                    {features.has('internal-catchall') ? project.slug : project.name}
```

Comment:
you don't need to change this because this is in `ProjectListOld` which we'll get rid of after we switch over: https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/views/organizationDashboard.jsx#L417

this already looks like this in production:
![screenshot 2018-03-06 11 23 14](https://user-images.githubusercontent.com/5026776/37053373-cedcf636-2130-11e8-8900-bbd89e587c08.png)